src/backend/src/routes/projects.routes.ts

```diff
@@ -47,5 +47,6 @@ projectRouter.post(
 projectRouter.post('/:wbsNum/set-team', nonEmptyString(body('teamId')), validateInputs, ProjectsController.setProjectTeam);
 projectRouter.delete('/:wbsNum/delete', ProjectsController.deleteProject);
 projectRouter.post('/:wbsNum/favorite', ProjectsController.toggleFavorite);
+projectRouter.post('/bom/material-type/create', nonEmptyString(body('name')), ProjectsController.createMaterialType);
```

Comment:
personally I like having BOMs in here. you could add a little comment denoting a 'BOM Section' kinda like we do for all the routes in the frontend. We could even make other sections within this routes file. I prefer the idea of a couple routes files with broad titles (like project, wp, finance) that relate directly to things on the frontend (so it's easily findable for new ppl) that are highly organized inside with comments/ sections rather than a bunch of smaller routes files that clearly relate to one feature or epic, but are rather disorganized inside (which is what we have rn)

Reply:
lmk thoughts @callmejim1226

Reply:
@RChandler234 Yeah I agree with your point bout this. I don't think we need to define separate routes for BOM in the backend since it's only being used in the project. Personally, it makes more sense to include all BOM backend endpoints in the project routes BOM section.